code/citadel/organs/genitals_sprite_accessories.dm

```diff
@@ -27,15 +27,57 @@
 	icon_state = "tapered"
 	name = "Tapered"
 
+//Vaginas
+/datum/sprite_accessory/vagina
+	icon = 'code/citadel/icons/vagina_onmob.dmi'
+	icon_state = null
+	name = "vagina"
+	gender_specific = 0
+	color_src = "vag_color"
+	locked = 0
+
+/datum/sprite_accessory/vagina/human
+	icon_state = "human"
+	name = "Human"
+
+/datum/sprite_accessory/vagina/gaping
+	icon_state = "gaping"
+	name = "Gaping"
+
+//I think this one is intended for arousal... but it would only work for human vaginas in this case...
+//So I'm leaving it as a selectable option until we can make/need variants for the sprites to change based on arousal.
+/datum/sprite_accessory/vagina/dripping
```

Comment:
Is there a difference between a sprite_accessory and an overlay?
My hope would be to have one basic and one aroused version for each type and use a genital boolean to check which one to use.
Although I think I could use place-holder copies for the ones missing images right now.